cg/cli/upload/delivery_report.py

```diff
@@ -50,10 +51,13 @@ def delivery_reports(context, print_console, force_report):
 
 @click.command("delivery-report")
 @click.argument("family_id", required=False)
-@click.option("-p", "--print", "print_console", is_flag=True, help="print report to console")
-@click.option("-f", "--force", "force_report", is_flag=True, help="overrule report validation")
+@click.option("-p", "--print", "print_console", is_flag=True, help="Print report to console")
+@click.option("-f", "--force", "force_report", is_flag=True, help="Overrule report validation")
+@click.option(
+    "--analysis-started-at", help="Use the analysis started at (i.e. 2020-05-28 " "12:00:46)"
```

Comment:
```suggestion
    "--analysis-started-at", help="Use the analysis started at (i.e.  '2020-05-28  12:00:46')"
```

Codefactor notified of this line, perhaps this can resolve it